src/boot/i18n.ts

```diff
@@ -21,11 +21,18 @@ declare module 'vue-i18n' {
 }
 /* eslint-enable @typescript-eslint/no-empty-interface */
 
+const getNavigatorLanguage = () => (navigator.languages && navigator.languages.length) ? navigator.languages[0] : (navigator as any).userLanguage || navigator.language || (navigator as any).browserLanguage || 'en-US';
```

Comment:
FYI Quasar has this utility:
https://quasar.dev/options/quasar-language-packs#detecting-locale